doc/howto/opam-file-generation.rst

```diff
@@ -113,9 +113,10 @@ General Notes and Tips
   keep the corresponding opam fields in a ``pkg.opam.template`` file. See
   :doc:`../reference/packages`.
 - It is not necessary to specify ``(version)``, this will be added at release
-  time if you use `dune-release`_.
-
-.. _dune-release: https://github.com/tarides/dune-release
+  time if you use `dune-release <https://github.com/tarides/dune-release>`_.
+- To generate an opam variable such as ``version``, use a colon ``:`` followed
```

Comment:
Do we have a page in the manual that specifies the variable behavior (such as `:with-test`, ...) and how they are used in dune? If yes, it could be useful to have a reference to it?
WDYT?

Reply:
there's https://github.com/ocaml/dune/blob/main/doc/explanation/opam-integration.rst which explains the dune/opam integration. we could add in as a general `..seealso::` at the end. sounds good?

Reply:
Yes, definitely :+1: 

However, I realized the first part of my suggestion is not relevant in this PR and should be address in another one. IIRC, there is no place (apart from examples) that `:with-test ` would be converted into `{with-test}` isn't it?

Reply:
Actually the `version` example is already in the reference for `(package)` but a xref doesn't hurt. I'll add both.